### PR TITLE
jquery-2.2.0 was removed from vendor/adminlete, changed to jquery-2.2.3

### PR DESCRIPTION
**Pull request by alashow**

### CODE REVIEW

**Comment by lloy0076** [comment]
It's at `2.2.4` now 🙂

File: src/resources/views/layout.blade.php [hunk 1]
```diff
@@ -102,7 +102,7 @@
 
     <!-- jQuery 2.2.0 -->
     <script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
-    <script>window.jQuery || document.write('<script src="{{ asset('vendor/adminlte') }}/plugins/jQuery/jQuery-2.2.0.min.js"><\/script>')</script>
+    <script>window.jQuery || document.write('<script src="{{ asset('vendor/adminlte') }}/plugins/jQuery/jQuery-2.2.3.min.js"><\/script>')</script>
```
Comment (lloy0076) [inline src/resources/views/layout.blade.php]: Ideally if we're going to do this, we'd actually look through the plugins directory for something that "looked like" jQuery although this does work, I suppose.

Comment (alashow) [reply src/resources/views/layout.blade.php]: or better: don't rely on AdminLTE and include it to backpack itself.

Comment (lloy0076) [reply src/resources/views/layout.blade.php]: https://thecodingmachine.github.io/discovery/

I don't really recomment it - it just seems to add even more complexity.

Comment (alashow) [reply src/resources/views/layout.blade.php]: I don't think there is a need for something like `discovery`. Just making sure the file is still there when updating AdminLTE or including it to backpack itself will do.

**Comment by tabacitu** [comment]
Merged, thank you!

**Pull request merged**